Tests/administrator/test_working.py

```diff
@@ -0,0 +1,42 @@
+# Third Party Imports
```

Comment:
I find these comments redundant and making below imports harder to read. Are they even required?

Reply:
Yes, they will be required while adding the tests in future but we can remove it for now.

Reply:
I can remove them, but it is a good practice to separate the type of imports namely (third-party, local-project, django)